ResidentInformationApi/V1/Factories/ResponseFactory.cs

```diff
@@ -0,0 +1,55 @@
+using System.Collections.Generic;
+using System.Linq;
+using ResidentInformationApi.V1.Boundary.Responses;
+using ResidentInformationApi.V1.Domain;
+using Address = ResidentInformationApi.V1.Domain.Address;
+using AddressResponse = ResidentInformationApi.V1.Boundary.Responses.Address;
+using HousingResidentInformation = ResidentInformationApi.V1.Domain.HousingResidentInformation;
+using HousingResidentInformationResponse = ResidentInformationApi.V1.Boundary.Responses.HousingResidentInformation;
+
+namespace ResidentInformationApi.V1.Factories
+{
+    public static class ResponseFactory
+    {
+        public static HousingResidentInformationResponse ToResponse(this HousingResidentInformation domain)
+        {
+            return new HousingResidentInformationResponse
```

Comment:
The tests for this are commented out I think? Probably best to have the tests for all of these methods

Reply:
Absolutely - apologies @LBHBReynolds-Carr, my fault entirely for leaving the tests commented-out when handing over this branch.